src/vs/workbench/contrib/terminal/browser/terminalService.ts

```diff
@@ -293,8 +292,7 @@ export class TerminalService implements ITerminalService {
 		}
 		reconnectedPromise.then(() => {
 			this._setConnected();
-			this._mark('code/terminal/didReconnect');
-			this._timerService.setPerformanceMarks('terminal', this._perfMarks);
+			this._mark('terminal/didReconnect');
```

Comment:
It's fine to roll your own thing but you aren't included in perf telemetry unless you use `ITimerService#setPerformanceMarks`. So, I would rather do `perf('code/terminal/...')` and later `ITimerService#getPerformanceMarks`

Reply:
Good to know, I've also got localPtyHost/remotePtyHost raw marks coming which use that to track those procs